docs/user-guide/thirdpartyresources.md

```diff
@@ -0,0 +1,118 @@
+
+```
+$ kubectl explain thirdpartyresource
+```
+
+## Creating a ThirdPartyResource
+
+When you user create a new `ThirdPartyResource`, the Kubernetes API Server reacts by creating a new, namespaced RESTful resource path. For now, non-namespaced objects are not supported. As with existing built-in objects, deleting a namespace deletes all custom objects in that namespace. `ThirdPartyResources` themselves are non-namespaced and are available to all namespaces.
+
+For example, if a save the following `ThirdPartyResource` to `resource.yaml`:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
+metadata:
+  name: cron-tab.stable.example.com
+description: "A specification of a Pod to run on a cron style schedule"
+versions:
+- name: v1
+```
+
+And create it:
+
+```shell
+$ kubectl create -f resource.yaml
+thirdpartyresource "cron-tab.stable.example.com" created
+```
+
+Then a new RESTful API endpoint is created at:
+
+`/apis/stable.example.com/v1/namespaces/<namespace>/crontabs/...`
+
+This endpoint URL can then be used to create and manage custom objects.
+
+## Creating Custom Objects
+
+After the `ThirdPartyResource` object has been created you can create custom objects. Custom objects can contain custom fields. These fields can contain arbitrary JSON. 
+In the following example, a `cronSpec` and `image` custom fields are set to the custom `CronTab` object. If you save the following YAML to `my-crontab.yaml`:
+
+```yaml
+apiVersion: "stable.example.com/v1"
+kind: CronTab
+metadata:
+  name: my-new-cron-object
+cronSpec: "* * * * /5"
+image: my-awesome-cron-image
+```
+
+and create it:
+
+```shell
+$ kubectl create -f my-crontab.yaml
+crontab "my-new-cron-object" created
+```
+
+You can then manage our `CronTab` objects using kubectl. Note that resource names are not case-sensitive when using kubectl:
+
+```shell
+$ kubectl get crontab
+NAME                 LABELS    DATA
+my-new-cron-object   <none>    {"apiVersion":"stable.example.com/v1","cronSpec":"...
+```
+
+You can also view the raw JSON data. Here you can see that it contains the custom `cronSpec` and `image` fields from the yaml you used to create it:
+
+```yaml
+$ kubectl get crontab -o json
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {},
+    "items": [
+        {
+            "apiVersion": "stable.example.com/v1",
+            "cronSpec": "* * * * /5",
+            "image": "my-awesome-cron-image",
+            "kind": "CronTab",
+            "metadata": {
+                "creationTimestamp": "2016-09-29T04:59:00Z",
+                "name": "my-new-cron-object",
+                "namespace": "default",
+                "resourceVersion": "12601503",
+                "selfLink": "/apis/stable.example.com/v1/namespaces/default/crontabs/my-new-cron-object",
+                "uid": "6f65e7a3-8601-11e6-a23e-42010af0000c"
+            }
+        }
+    ]
+}
+```
```

Comment:
Mention that this example doesn't actually give you cron job support.  You'd still need to write a controller object that looked at cronTabs and did something useful.  But you do get durable storage of the cronTab objects.